spec/helpers/ruby_version_spec.rb

```diff
@@ -103,11 +89,39 @@
     end
   end
 
-  it "detects Ruby 2.0.0" do
-    Hatchet::App.new("mri_200").in_directory_fork do |dir|
+  it "detects Ruby from Gemfile.lock" do
+    Hatchet::App.new("default_ruby").in_directory_fork do |_|
+      dir = Pathname(Dir.pwd)
       Bundler.with_unbundled_env do
+        dir.join("Gemfile").write(<<~EOF)
+          source "https://rubygems.org"
+
+          gem 'rake'
+          ruby '3.2.3'
+        EOF
+        dir.join("Gemfile.lock").write(<<~EOF)
+          GEM
+            remote: https://rubygems.org/
+            specs:
+              rake (13.2.1)
+
+          PLATFORMS
+            arm64-darwin-22
+            ruby
+            x86_64-linux
+
+          DEPENDENCIES
+            rake
+
+          RUBY VERSION
+            ruby 3.2.3p157
+
+          BUNDLED WITH
+            2.4.19
+        EOF
+
         ruby_version   = LanguagePack::RubyVersion.new(@bundler.install.ruby_version, is_new: true)
-        version_number = "2.0.0"
```

Comment:
Update test to use a modern Ruby version and move away from the less flexible github based fixture. Write the Gemfile and Gemfile.lock instead so the developer can see what's being used in the test at a glance.